components/Shared/MnemonicWord/InputWord.jsx

```diff
@@ -15,13 +15,14 @@ import contentProps from './contentProps'
 
 const setBackgroundColor = ({ completed, empty, valid, importSeedError }) => {
   if (importSeedError && (empty || !valid)) return 'status.fail.background'
-  if (!importSeedError && completed) return 'core.primary'
+  if (!importSeedError && completed) return 'core.secondary'
   if (!importSeedError && empty) return 'core.white'
-  return 'core.white'
+  return 'core.secondary'
 }
 
-const setInputColor = props => {
-  if (props.completed) return 'core.white'
+const setInputColor = ({ completed, importSeedError }) => {
+  if (importSeedError) return 'core.nearblack'
+  if (!importSeedError && completed) return 'core.primary'
   return 'core.primary'
 }
```

Comment:
@Schwartz10 Used the props from `setBackgroundColor` to get more granular in targeting, here. 

But it doesn't look like I'm setting the correct condition, as I cannot get `core.nearblack` to render per my example screenshot:

<img width="925" alt="Screen Shot 2020-04-14 at 1 18 12 PM" src="https://user-images.githubusercontent.com/6787950/79249221-53b55700-7e53-11ea-86ca-c148dad01840.png">

Reply:
https://github.com/openworklabs/filecoin-web-wallet/pull/384/commits/ba07e5f947f196964514be47acd94955c19e612c